screenapiexample/src/main/java/net/gini/android/vision/screen/MainActivity.java

```diff
@@ -92,6 +92,10 @@ private void startGiniVisionLibrary() {
         // Set EXTRA_IN_SHOW_ONBOARDING to true, to show the OnboardingActivity when the CameraActivity starts
         //intent.putExtra(CameraActivity.EXTRA_IN_SHOW_ONBOARDING, true);
 
+        // Set EXTRA_IN_BACK_BUTTON_SHOULD_CLOSE_LIBRARY to true, to close library on pressing the back
+        // button from any Activity in the library
+        //intent.putExtra(CameraActivity.EXTRA_IN_BACK_BUTTON_SHOULD_CLOSE_LIBRARY, true);
```

Comment:
Having so many configurations it could make sense to create a builder to generate the intent. Or maybe at least static factory methods. Then the static strings can be hidden and no one has to write the above code 😛